exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
 
-console.log(message);
+let name = "Nesrin";
+let nameLength = name.length;
```

Comment:
Same consideration here as above, prefer to use `const` rather than `let` whenever possible.